src/rp2_common/pico_platform/include/pico/platform.h

```diff
@@ -138,14 +138,14 @@ extern "C" {
  * Data marked this way will retain its value across a reset (normally uninitialized data - in the .bss
  * section) is initialized to zero during runtime initialization
  *
- * For example a `uint32_t` foo will retain its value if the program is restarted
+ * For example a `uint32_t` foo that will retain its value if the program is restarted by reset.
  *
  *     uint32_t __uninitialized_ram("my_group_name") foo;
  *
  * The section attribute is `.uninitialized_ram.<group>`
  *
  * \param group a string suffix to use in the section name to distinguish groups that can be linker
- *              garbage collected independently
+ *              garbage-collected independently
```

Comment:
You've changed a couple of "garbage collected" -> "garbage-collected" but there's still a bunch that say "garbage collected" ?